composer.json

```diff
@@ -4,7 +4,7 @@
     "keywords": ["php-pusher-server", "pusher", "rest", "realtime", "real-time", "real time", "messaging", "push", "trigger", "publish", "events"],
     "license": "MIT",
     "require": {
-        "php": "^7.3|^8.0",
+        "php": "^7.3|^8.0|^8.1|^8.2",
```

Comment:
`|^8.1|^8.2` is incorrect. `^8.0` already allows all 8.x.

Reply:
i would recommend just reverting the change to the composer.json. people can install it on any php 8 version, at their own risk. that's super common.